RealEstate/apps/core/views.py

```diff
@@ -9,31 +9,31 @@
 
 
 def login(request, *args, **kwargs):
-    """
-    If the user is already logged in and they navigate to the login URL,
-    just redirect them home. Otherwise just delegate to the default
-    Django login view.
-    """
-    if request.user.is_authenticated():
-        return redirect(reverse('home'))
-    return auth_login(request, *args, **kwargs)
+	"""
+	If the user is already logged in and they navigate to the login URL,
+	just redirect them home. Otherwise just delegate to the default
+	Django login view.
+	"""
+	if request.user.is_authenticated():
+		return redirect(reverse('home'))
+	return auth_login(request, *args, **kwargs)
 
 
 class BaseView(View):
-    """
-    All subclassed views will redirect to the login view if not logged in.
-    """
-    @method_decorator(login_required)
-    def dispatch(self, request, *args, **kwargs):
-        return super(BaseView, self).dispatch(request, *args, **kwargs)
+	"""
+	All subclassed views will redirect to the login view if not logged in.
+	"""
+	@method_decorator(login_required)
+	def dispatch(self, request, *args, **kwargs):
+		return super(BaseView, self).dispatch(request, *args, **kwargs)
 
 
 class HomeView(BaseView):
-    """
-    View for the home page, which should render different templates based
-    on whether or not the the logged in User is a Realtor or Homebuyer.
-    """
-    def get(self, request, *args, **kwargs):
-        couple = Couple.objects.filter(homebuyer__user=request.user)
-        house = House.objects.filter(couple=couple)
-        return render(request, 'core/homebuyerHome.html', {'couple': couple, 'house': house})
+	"""
+	View for the home page, which should render different templates based
+	on whether or not the the logged in User is a Realtor or Homebuyer.
+	"""
+	def get(self, request, *args, **kwargs):
+		couple = Couple.objects.filter(homebuyer__user=request.user)
+		house = House.objects.filter(couple=couple)
+		return render(request, 'core/homebuyerHome.html', {'couple': couple, 'house': house})
```

Comment:
So I was wondering why this file was committed when nothing has changed semantically, and it looks like tab characters are being inserted instead of 4 spaces for indentation.  In python this is very bad to mix tabs and spaces for indentation because python actually relies on the indentation for control flow.  Lets stick with the pep8 convention for using 4 spaces for each indentation level.

https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

Baiyu I am assuming you are using the Idea IDE based on the .idea file, can you configure your editor to replace all these tab characters with 4 spaces?  Thanks.